contracts/linear/src/liquidity_pool.rs

```diff
@@ -443,6 +443,44 @@ impl LiquidStakingContract {
         results.iter().map(|amount| (*amount).into()).collect()
     }
 
+    /// As we're going to completely deprecate liquidity pool, this function allows
+    /// removing all liquidity from users and return NEAR and LiNEAR back to users
+    pub fn remove_all_liquidity(&mut self, account_id: AccountId) -> Vec<U128> {
+        self.assert_running();
```

Comment:
Should we also assert predecessor, this seems permissionless?

Reply:
The purpose of the function is for returning all liquidity back to the providers, so we don't require the provider to call the function.